### DISCUSSION TITLE
Can it be 1 pass with 3 proofs?

### DISCUSSION THREAD

**Discussion by alexwwstx** [Chia-Network/chia-blockchain#7654] [Support]
log shows 1 plot is passed, and 3 proofs are found, can it be?
Does it mean the proof can be repeated in a single plot?

here's the log
2021-07-15T03:31:11.388 harvester chia.harvester.harvester: INFO     1 plots were eligible for farming b32be96f36... Found 3 proofs. Time: 0.97034 s. Total

**Comment by ktkiebel** [comment]
I just saw the same thing on my harvester with no chia reward yet.

**Reply by alexwwstx** [reply]
mine is in a pool, expected to see no reward yet. My question is how it can have multiple proofs in a single passed plot, does it mean there are duplicated records in the same plot file, is it expected or a bug?

**Reply by gryan315** [reply]
each plot contains hundreds of thousands of possible proofs. The partial proof is only part of the full proof, so there is a small chance that one plot will contain multiple combinations of the same partial proof.

**Reply by alexwwstx** [reply]
Just a curious question, if it happens in self farming, does it count once to get 2 XCH or 6 XCH?

**Reply by gryan315** [reply]
I doubt a plot could contain 3 full proofs similar enough to win. The partial proofs are matches of a smaller set of numbers so there are millions and millions of potential partial proofs in each plot.
